frontend/public/components/image-stream.tsx

```diff
@@ -167,17 +167,17 @@ export const ExampleDockerCommandPopover: React.FC<ImageStreamManipulationHelpPr
         <div>
           <p>
             {t(
-              'public~Create a new Image Stream Tag by pushing an image to this Image Stream with the desired tag.',
+              'public~Create a new ImageStream Tag by pushing an image to this ImageStream with the desired tag.',
```

Comment:
The first one should probably be ImageStreamTag; it looks like that's a specific resource kind: https://github.com/openshift/console/blob/master/frontend/public/models/index.ts#L326

Reply:
Made change to ImageStreamTag.